addons/sourcemod/scripting/FamilyShareManager.sp

```diff
@@ -477,8 +477,27 @@ public void SteamWorks_OnValidateClient(int ownerauthid, int authid)
 
     int client = GetClientOfAuthId(authid);
 
+    bool whiteListed = false;

```

Comment:
code is exactly the same as in https://github.com/srcdslab/sm-plugin-FamilyShareManager/blob/bbed9015aeea946eb85dbe05e2942c202daa1b52/addons/sourcemod/scripting/FamilyShareManager.sp#L352

so put it in a function to stay DRY:
- https://deviq.com/principles/dont-repeat-yourself